src/edit/DistortableImage.EditToolbar.js

```diff
@@ -1,162 +1,163 @@
 L.DistortableImage = L.DistortableImage || {};
 
 var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({
-		initialize: function(map, overlay, options) {
-			this._overlay = overlay;
-			this._map = map;
-
-			LeafletToolbar.ToolbarAction.prototype.initialize.call(this, options);
-		}
-	}),
-
-	ToggleTransparency = EditOverlayAction.extend({
-		options: { toolbarIcon: {
-			html: '<span class="fa fa-adjust"></span>',
-			tooltip: 'Toggle Image Transparency',
-			title: 'Toggle Image Transparency'
-		}},
-
-		addHooks: function() {
-			var editing = this._overlay.editing;
-
-			editing._toggleTransparency();
+    initialize: function(map, overlay, customAction, options) {
```

Comment:
This looks good, but where is it called from, and how does `customAction` relate to the submenus? Is it related to this PR? I think perhaps we need to see this in a test to better understand how it works; would you mind adding the test to this PR? Thank you!

Reply:
@jywarren `customAction` offers a deeper degree of control over some actions, which I thought was actually unnecessary for this PR right now, so I've removed it. I'll reintroduce it if we feel the need to, along with the docs.